core/services/ocr2/plugins/ccip/internal/ccipdata/usdc_reader_internal_test.go

```
@@ -28,7 +29,7 @@ func TestLogPollerClient_GetLastUSDCMessagePriorToLogIndexInTx(t *testing.T) {
 
 	t.Run("multiple found", func(t *testing.T) {
 		lp := lpmocks.NewLogPoller(t)
-		u := NewUSDCReader(lggr, utils.RandomAddress(), lp)
+		u, _ := NewUSDCReader(lggr, "job_123", utils.RandomAddress(), lp, false)
```

Should we have `require.NotError` on currently ignored error?

---

Yes